Sources/PenguinStructures/Graphs/AdjacencyList.swift

```diff
@@ -69,6 +70,7 @@ public struct AdjacencyList<IdType: BinaryInteger>: GraphProtocol {
 		assert(id.id < edgesArray.count, "Vertex \(id)\(makeName()) is not valid.")
```

Comment:
Might want to leave this for tomorrow, but consider whether this does what you want in release builds.  If it's just a sanity check for the library, you got the right call, but from your use of the word “precondition” in comments below it might not be.